x-pack/test/security_solution_api_integration/test_suites/detections_response/utils/rules/rule_gaps.ts

```diff
@@ -36,5 +42,118 @@ export const scheduleRuleRun = async (
     .set('x-elastic-internal-origin', 'Kibana')
     .send(params)
     .expect(expectedStatusCode);
+
+  return response.body;
+};
+
+export const findAllBackfills = async (supertest: SuperTest.Agent, expectedStatusCode = 200) => {
+  let total = 0;
+  let page = 1;
+  const perPage = 100;
+  let backfills: FindBackfillResponse['body']['data'] = [];
+  // ?page=1&per_page=100&sort_field=createdAt&sort_order=desc
+  while (backfills.length < total || total === 0) {
+    const response: FindBackfillResponse = await supertest
```

Comment:
`FindBackfillResponse` fetching could be moved to a separate function for better modularity and this new function could be shared with `findBackfills`.